cli/src/main/java/org/jboss/pnc/bacon/cli/App.java

```diff
@@ -166,7 +164,9 @@ public static void main(String[] args) throws Exception {
         try {
             App app = new App();
             System.exit(app.run(args));
-        } catch (FatalException e) {
+        } catch (FatalException ex) {
+            log.error(ex.getMessage());
+            log.debug("Full trace", ex);
```

Comment:
what do users have to do to get to the full stack trace? Is debug printed to some log file? Or do we have to use some --verbose option or sth?

Reply:
The theory _should_ be that enabling verbose should give the debug logging. I suspect there is a limitation in aesh (hence my suggestion in NCL-5974 to consider picocli) that it can't handle enabling verbose and then handling other options which might throw an exception.

Reply:
pig had an execution.log that contained all logging info, it was printed at the end of a product job if it failed. I'll look into adding such thing for bacon

Reply:
I think he suggested standardizing on `e` ;)

Reply:
The log would need to be captured by cpaas.